dotCMS/src/main/java/com/dotmarketing/portlets/containers/transform/ContainerTransformer.java

```diff
@@ -0,0 +1,61 @@
+package com.dotmarketing.portlets.containers.transform;
+
+import com.dotcms.util.transform.DBTransformer;
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.portlets.containers.model.Container;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * DBTransformer that converts DB objects into Container instances
+ */
+public class ContainerTransformer implements DBTransformer {
+    final List<Container> list;
+
+
+    public ContainerTransformer(List<Map<String, Object>> initList){
+        List<Container> newList = new ArrayList<>();
+        if (initList != null){
+            for(Map<String, Object> map : initList){
+                newList.add(transform(map));
+            }
+        }
+
+        this.list = newList;
+    }
+
+    @Override
+    public List<Container> asList() throws DotStateException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)